### PR TITLE
I#3215

### PR DESCRIPTION
**Pull request by j0rt3g4**
Updated Image with white background.
#3215

### CODE REVIEW

**Comment by lindspea** [comment]
@scanum

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/healthcare/amion-api.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+search.appverid: MET150
+localization_priority: Normal
+MS.collection: Teams_ITAdmin_PracticalGuidance
+appliesto:
+- Microsoft Teams
+ms.reviewer: vanido, acolonna
+description: scheduling interop with shifts
+---
+
+# AMION scheduling app for Teams
```
Comment (JohanFreelancer9) [inline Teams/healthcare/amion-api.md]:
```suggestion
# Amion scheduling app for Teams 
```

File: Teams/healthcare/amion-api.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+MS.collection: Teams_ITAdmin_PracticalGuidance
+appliesto:
+- Microsoft Teams
+ms.reviewer: vanido, acolonna
+description: scheduling interop with shifts
+---
+
+# AMION scheduling app for Teams 
+
+The AMION app displays on-call data for hospitals and gives users an option to import that data into Shifts. The import happens seamlessly using the Graph APIs without further user input.
```
Comment (JohanFreelancer9) [inline Teams/healthcare/amion-api.md]:
```suggestion
The Amion app displays on-call data for hospitals and gives users an option to import that data into Shifts. The import happens seamlessly using the Graph APIs without further user input. 
```

File: Teams/healthcare/amion-api.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## FeatureName overview
+
+Short overview of what the new feature is. 
+
+- What is the new or updated experience?
+
+- Does this feature replace an existing feature/experience? If yes, what is the transition plan?
+
+- Does this feature has dependency on other features? If yes, list/explain the dependencies.
```
Comment (JohanFreelancer9) [inline Teams/healthcare/amion-api.md]:
```suggestion
- Does this feature have dependency on other features? If yes, list/explain the dependencies.
```

File: Teams/healthcare/amion-api.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+- List the key deployment scenarios - why would people use this feature? 
+
+## [OPTIONAL] Planning for feature
+
+Some features would require careful pre-planning before deployment can begin. For those features, cover what planning tasks customers should complete.
```
Comment (JohanFreelancer9) [inline Teams/healthcare/amion-api.md]:
```suggestion
Some features would require careful pre-planning before deployment can begin. For those features, cover the planning tasks that customers should complete.
```

File: Teams/healthcare/amion-api.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+- Is this feature per org or per user? 
+
+- Use PowerShell or UI?
+
+- Short instructions on how to turn on the feature for the organization or for individual users. 
+
+## Manage feature
+
+- Add subsections with basic management tasks that would be required or that we expect customer to perform.
```
Comment (JohanFreelancer9) [inline Teams/healthcare/amion-api.md]:
```suggestion
- Add subsections with basic management tasks that is required or that the customer is expected to perform. 
```

**Comment by lindspea** [comment]
@j0rt3g4 please check conflict that needs to be resolved. I can't edit.

**Comment by scanum** [comment]
@j0rt3g4 Has this been solved? Thanks!

**Comment by j0rt3g4** [comment]
@scanum Yeah it's the additional one I've commented to you over teams

**Comment by scanum** [comment]
@j0rt3g4 @lindspea Can we change the author to Ken? Thanks!

**Comment by j0rt3g4** [comment]
@scanum  sure.

**Comment by scanum** [comment]
@kenwith PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by SerdarSoysal** [comment]
There is a brand new topic in this PR. I have to get PM sign-off for this.

**Comment by j0rt3g4** [comment]
Substituted this by #3488 
I've fixed the references to that new topic that was on this PR for whatever reason.

@SerdarSoysal   @scanum @lindspea  
Please remit to the new PR #3488  Thank you!.
This will be way clearer and easier to merge and solve this issue. #3215

**Pull request closed without merging**